tests/test_ssl.py

```diff
@@ -101,6 +101,21 @@
 skip_if_py3 = pytest.mark.skipif(PY3, reason="Python 2 only")
 
 
+def socket_any_family():
+    try:
+        return socket(AF_INET)
+    except error:
+        return socket(AF_INET6)
+
+
+def loopback_address(socket):
+    if socket.family == AF_INET:
+        return "127.0.0.1"
+    if socket.family == AF_INET6:
+        return "::1"
+    raise ValueError("Unknown socket family")
```

Comment:
To make coverage happy you can write this:

```
if socket.family == AF_INET:
    return "127.0.0.1"
else:
    assert socket.family == AF_INET6
    return "::1"
```

my kingdom for proper enums 🤷‍♂️

Reply:
Done.